physionet-django/console/templates/console/guidelines_course.html

```diff
@@ -0,0 +1,161 @@
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  }]
+}
+      </pre>
+        <ol start="1">
+          <li><strong>Course Information:</strong> Fill out all fields in the JSON file with the appropriate information
+            about your course, including:
+          </li>
+          <ul>
+            <li>The name of the course</li>
+            <li>A description of the course in html</li>
+            <li>The valid duration of the course</li>
+            <li>The version of the course</li>
+          </ul>
+          <small><strong>Note the importance of versioning:</strong> It's essential to keep track of the version number
+            of your course.
+            Please make sure the version number is unique for each course update. Even minor changes, such as a spelling
+            correction,
+            require an update to the version number in the new JSON file (refer to the create and update JSON examples
+            above).
+            Please use the Semantic Versioning system (Major.Minor) for version numbers.
+            If you update a course's major version (e.g., from 1.5 to 2.0), all existing training certificates provided
+            to
+            users will expire after a month. To regain access to resources, users must retake the new version of the
+            training.</small>
+
+        </ol>
+        <ol start="2">
+          <li><strong>Modules:</strong> A course may have one or more modules that contain the contents,quizzes for each
+            section of the course. Each module should include:
+          </li>
+          <ul>
+            <li>A name for the module</li>
+            <li>A description of the module in html</li>
+            <li>An order for the module</li>
+            <li>One or more content items in the module</li>
+            <li>One or more quizzes in the module</li>
+          </ul>
+        </ol>
+        <ol start="3">
+          <li><strong>Content:</strong> A module may have one or more content. Each content should include:</li>
+          <ul>
+            <li>A body for the content in html</li>
+            <li>An order for the content</li>
+          </ul>
+        </ol>
+        <ol start="4">
+          <li><strong>Quizzes:</strong> A module may have one or more quiz. Each Quiz should include:</li>
+          <ul>
+            <li>A question for the quiz</li>
+            <li>An order for the quiz</li>
+            <li>One or more choices for the quiz, including the correct answer</li>
+            <small><strong>Order:</strong> Note that the order is used to determine which content or quiz comes first,
+              so please
+              make sure you order the content and quiz properly. <br> For example, here the user will see the content
+              and quiz in the following order
+              content1, quiz 1, content 2, quiz 2.
+              <pre><code>
+"contents": [
+    {
+      "body": "content 1",
```

Comment:
I think it will be useful for the text in this example to give the reader an idea of how the content will flow. So my following suggestions aim towards having the text describe when/where the content will appear and how the course will actually function.


```suggestion
      "body": "This content will display first for the given module.",
```